taxclock/scrapers/base.py

```diff
@@ -63,23 +74,72 @@ def aws_store(self, data):
             data = self.sort_data_by_date(data)
             try:
                 self.s3.put_object(
+                    # TODO: Pull bucket from settings
                     Bucket=env.AWS_S3_BUCKET,
                     ACL='public-read',
                     Key='data/news.json',
                     Body=json.dumps(data[:7]))
             except Exception as err:
                 self.local_store(data[:7])
                 log.info(str(err))
+                self.error_details['error'] = str(err)
+                self.error_details['site_url'] =\
+                    "S3 is not available for all urls"
+                self.error_details['severity'] = env.ERROR_SEVERITY[1]
+                self.send_error_log(self.error_details)
         else:
             log.error('No data to save')
 
     def sort_data_by_date(self, data):
         '''Sorts data by date.
         Usage::
             Pass the data to sort.
-        :rtype: outputs the sorted data.
+        :rtype: outputs a list of sorted data.
         '''
         if not data:
             log.info('No data was found for sorting.')
         return sorted(data, key=lambda k: parse(k['date_published']),
                       reverse=True)
+
+    def format_error(self, error):
```

Comment:
Is there a way to turn this into a handler instead?

Reply:
Looking into this.